src/utils/utils.ts

```diff
@@ -114,4 +116,17 @@ export async function killProcessTree(pid: number, signal?: string | number) {
             }
         });
     });
+}
+
+export function getRandomOutputFolder() {
+    let randomFolder = Math.random().toString(36).substring(7);
+    return path.join(tempDirectory, randomFolder);
+}
+
+export function normalizeDriveLetter(filePath: string) {
+    if(process.platform === 'win32'){
+        return filePath.replace(/^([A-Z]):/, (match, driver) => `${driver.toLowerCase()}:`);
```

Comment:
why do we need to normalize driver letter?

Reply:
the `__dirname` is not stable for driver letter case

---
In reply to: [429754881](https://github.com/docascode/vscode-docs-build/pull/106#discussion_r429754881) [](ancestors = 429754881)